README.md

```diff
@@ -1,18 +1,48 @@
 # AlpacaProxy
 
-To start your Phoenix server:
+The Alpaca Proxy serves as a proxy between Belay and an alpaca account. If you plan to utilize Belay to offer insurance to your investors, and you utilize Alpaca as your brokerage provider, this proxy serves as a intermediatory between your Alpaca brokerage service and Belay. This proxy allows you to control what permissions we have to your Alpaca brokerage service and hide your private keys while still giving Belay a way to communicate to Alpaca.
 
-  * Run `mix deps.get` to install and setup dependencies
-  * Start Phoenix endpoint with `mix phx.server` or inside IEx with `iex -S mix phx.server`
+## Usage
 
-Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
+This application intends to be hosted on a hosting platform of your choice. For your convenience, we provide a docker package to allow for easy deployment
 
-Ready to run in production? Please [check our deployment guides](https://hexdocs.pm/phoenix/deployment.html).
+### ENVs required
 
-## Learn more
+There are some envs that will be required across any method of running AlpacaProxy, these include:
 
-  * Official website: https://www.phoenixframework.org/
-  * Guides: https://hexdocs.pm/phoenix/overview.html
-  * Docs: https://hexdocs.pm/phoenix
-  * Forum: https://elixirforum.com/c/phoenix-forum
-  * Source: https://github.com/phoenixframework/phoenix
+- `BELAY_SECRET`: This is a secret generated by the hoster and then shared with Belay to allow us access
+- `ALPACA_BASE_URL`: The Alpaca base url, for sandbox it will be: `https://broker-api.sandbox.alpaca.markets/`, for production, it will be: `https://broker-api.alpaca.markets/`
+- `ALPACA_KEY`: The Alpaca key retrieved from the Alpaca dashboard
+- `ALPACA_SECRET`: The Alpaca secret retrieved from the Alpaca dashboard
+
+### Via Docker
```

Comment:
This all looks good. Can you just mention that docker will need to be listening on port 4000, that's part of what they'll have to configure to run it.